raiwidgets/raiwidgets/responsibleai_dashboard.py

```diff
@@ -66,6 +66,13 @@ def causal_whatif():
 
         self.add_url_rule(causal_whatif, '/causal_whatif', methods=["POST"])
 
+        def causal_global_effects():
```

Comment:
should we add some dropdown in the UI to choose the cohort?  I know the other components either have a dropdown (like dataset explorer) or allow the user to see the cohorts side by side (model performance, feature importances):
![image](https://user-images.githubusercontent.com/24683184/178791455-d74bf399-45cc-4b84-8431-568a3b359f0b.png)

Reply:
The PR just adds the SDK end points for global effects and global policy. The way I am thinking about these would work in UI would be through global selected cohort. What do you think?